client/src/components/StreamField/blocks/StructBlock.js

```diff
@@ -111,6 +111,19 @@ export class StructBlock {
     return state;
   }
 
+  getDuplicatedState() {
+    const state = {};
+    // eslint-disable-next-line guard-for-in
+    for (const name in this.childBlocks) {
+      const block = this.childBlocks[name];
+      state[name] =
+        block.getDuplicatedState === undefined
```

Comment:
Looks like we are missing a test addition for this component?

Reply:
This was sort of covered in StreamBlock.test.js but I've split them out to handle the block types more specifically in their corresponding modules.

Reply:
👍